README.md

```diff
@@ -55,6 +70,18 @@ Otherwise, your script for executing specs may look something like:
 chmod +x test-splitter
```

Comment:
This bit can be deleted if we're recommending the user to run the test splitter directly via the step command